imod/prepare/topsystem/allocation.py

```diff
@@ -394,7 +404,8 @@ def _allocate_cells__first_active_to_elevation(
 
     top_layered = _enforce_layered_top(top, bottom)
 
-    riv_cells = (upper_active_layer <= layer) & (bottom_elevation < top_layered)
+    above_lower_bound = _is_above_lower_bound(bottom_elevation, top_layered)
```

Comment:
could you rename the variable `above_lower_bound` to `is_above_lower_bound`. Making boolean variables start with `is` or `has` makes the code easier to read